platforms/taobao-minigame/wrapper/builtin/XMLHttpRequest.js

```diff
@@ -90,61 +90,25 @@ export default class XMLHttpRequest extends EventTarget {
     if (this.readyState !== XMLHttpRequest.OPENED) {
       throw new Error("Failed to execute 'send' on 'XMLHttpRequest': The object's state must be OPENED.")
     } else {
-      let myRequestTask = my.request({
-        data,
+      let myRequestTask = my.tb.request({
+        body: JSON.stringify(data),
```

Comment:
is `data` typeof `object`, it seems data is typeof `string`  
why we use JSON.stringify here

Reply:
body require a json string